editor/RubyScanner.cpp

```diff
@@ -140,7 +140,7 @@ bool Scanner::didBlockStart()
 
 bool Scanner::didBlockEnd()
 {
-    static const QRegExp regex(QLatin1String("24_"));
+    static const QRegExp regex(QLatin1String("(24|29)_"));
```

Comment:
This:
```
if a == 4; if b == 8
  do_something
else
  do_something_else
end
and_another_thing
end
```

Not a big deal because this kind of code style isn't used and avoided by the Ruby community, however it need to be fixed for the sake of correctness.